app/components/filters/postal-select.js

```diff
@@ -33,7 +33,18 @@ export default class PostalCodeMultipleSelectComponent extends Component {
 
   @task
   *loadPostalCodes() {
-    this.postalCodes = yield this.store.findAll('postal-code');
+    this.postalCodes = yield this.store.query('postal-code', {
+      page: { size: 100 },
+      sort: ':no-case:postal-code',
+    });
+
     this.args.onChange(this.selectedPostalCodes());
   }
+
+  searchMethod(term, select) {
+    return select.options.filter(
+      (item) =>
+        item.postalCode.includes(term) || item.postalName.includes(term),
```

Comment:
@oscartison maybe you can change this filter to check on lowercase?
For example If I search for baa it's not giving results. But if I search for Baa it will give me the correct result Baardegem.